### ISSUE TITLE
Add support for custom linters

### ISSUE THREAD

**Issue by marcogallotta** [brigade/scss-lint#161]
Hey there

I am trying to add custom linters. The code we use to add our linters is included below. It was working in scss-lint version 0.12.0, but not on the latest version. We are enabling it by passing the args "-i <linter-name>", and the linters are in admin/lint/scss_lint_rules/ and follow the same structure as pre-packaged ones. It just never gets loaded into @options.

Any suggestions on how to get this working? We have also tried enabling it through a config file, but that doesn't seem to do anything.

```ruby
    require 'scss_lint'
  
    ASANA_LINT_DIR = "admin/lint/scss_lint_rules/"
  
    SCSSLint::LinterRegistry.class_eval do
      class << self
        def extract_linters_from(linter_names)
          puts "extracting linters"
          Dir["#{ASANA_LINT_DIR}*.rb"].each do |file|
            require file
          end
          linter_names.map do |linter_name|
            begin
              SCSSLint::Linter.const_get(linter_name)
            rescue NameError
              raise SCSSLint::NoSuchLinter.new("Linter #{linter_name} does not exist")
            end
          end
        end
      end
    end

    SCSSLint::CLI.new(ARGV).tap do |cli|
      puts "parsing args"
      cli.parse_arguments
      puts "running"
      cli.run
    end
```

and - any chance you'd add the functionality to pass custom linters in?  I'm sure lots of other groups have custom lint rules that don't make sense to add to this package, but would be of great use to them.

thanks!!

**Comment by gunnx** [comment]
+1 for ForbiddenProperty

**Comment by sds** [comment]
Experimental support for custom linters was implemented in #459. Please check out the `master` branch to take it for a spin. Feedback would be appreciated!
